package.json

```diff
@@ -72,10 +72,8 @@
     "express": "^4.17.1",
     "front-matter": "^4.0.0",
     "fs-extra": "^10.0.0",
-    "glob": "^7.1.5",
```

Comment:
This was added in 8b8d94c78a9c996c2da6916b2e883c6bb7f0641d for `tasks/transform-types.js`.  It should have been removed in 7c6755d3ecca774a332390c1e271bac5c0f363a9 when that task was removed.